kernel/arch/dreamcast/include/dc/vmu_fb.h

```diff
@@ -0,0 +1,122 @@
+/* KallistiOS ##version##
+
+   dc/vmu_fb.h
+   Copyright (C) 2023 Paul Cercueil
+
+*/
+
+#include <kos.h>
```

Comment:
Please do not include <kos.h> in any public header files. Include the appropriate header files that you need, please.